src/core/templates/navigation.hbs

```diff
@@ -1,5 +1,8 @@
 {{! All links in the nav bar should have an attritbue of data-event, navigationView uses this to fire a global event}}
 <div class="navigation-inner clearfix" role="navigation" aria-label="{{_globals._accessibility._ariaLabels.navigation}}">
+    {{#if _accessibility._isSkipNavigationEnabled}}
+        <button class="skip-nav-link" aria-label="Skip navigation" data-event="skipNavigation" role="button">Skip Navigation</button>
```

Comment:
It doesn't seem right that "Skip Navigation" is hard-coded twice here.  

We need to add a language string to the `_globals._accesibility` object, like `_accessibilityToggleTextOn` and another for the aria-label.